sdk/resourcemanager/Azure.ResourceManager.Core/src/Azure.ResourceManager.Core.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>1.0.0-beta.1</Version>
+    <PackageId>Azure.ResourceManager.Core</PackageId>
+    <Description>
+      Azure management core SDK for Azure resources.
+      This is a beta preview vesion.  This version uses a next-generation code generator that introduces important breaking changes, but also new features (such as intuitive authentication, custom HTTP pipeline, distributed tracing and much more).
+    </Description>
+    <PackageTags>azure;management;resource</PackageTags>
+    <NoWarn>$(NoWarn);AZC0008;AZC0001;AZC0107;CA2214;CA1036;CA1067;CA1065;SA1028</NoWarn>
+    <IncludeGeneratorSharedCode>true</IncludeGeneratorSharedCode>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Collections.Immutable" />
```

Comment:
Does our Azure.Core dependency come through Azure.Identity?

I assume we are not fully integrating into their targets and proeprties files yet.

Reply:
We are taking dependency thru build.props that ended up here. [https://github.com/Azure/azure-sdk-for-net/blob/d1e5620b9bce5422892b129de43182507dfc0bdf/sdk/core/Azure.Core/src/Azure.Core.props#L4](https://github.com/Azure/azure-sdk-for-net/blob/d1e5620b9bce5422892b129de43182507dfc0bdf/sdk/core/Azure.Core/src/Azure.Core.props#L4)

---
In reply to: [583344699](https://github.com/Azure/azure-sdk-for-net/pull/18968#discussion_r583344699) [](ancestors = 583344699)